kotlin-audio/src/main/java/com/doublesymmetry/kotlinaudio/notification/NotificationManager.kt

```diff
@@ -263,35 +271,44 @@ class NotificationManager internal constructor(private val context: Context, pri
     override fun onNotificationPosted(notificationId: Int, notification: Notification, ongoing: Boolean) {
         scope.launch {
             event.updateNotificationState(NotificationState.POSTED(notificationId, notification, ongoing))
+
+            reload()
         }
     }
 
     override fun onNotificationCancelled(notificationId: Int, dismissedByUser: Boolean) {
         scope.launch {
             event.updateNotificationState(NotificationState.CANCELLED(notificationId))
+
+            reload()
         }
```

Comment:
I started calling the `reload` function in almost every event, so it forces the notification to re-render, but that caused it to misbehave even more. It made me wonder if the whole function is bad to use in general. More info about this in the PR description.